### PR TITLE
[Docs Site] Fix footnotes with multiple references

### PR DESCRIPTION
**Pull request by KianNH**
### Summary

Fix footnotes with multiple references

### CODE REVIEW

**Review by Oxyjun — APPROVED** [review]

**Comment by GregBrimble** [comment]
Got an example of where this is used?

**Comment by KianNH** [comment]
> Got an example of where this is used?

https://developers.cloudflare.com/durable-objects/platform/limits/ vs. https://kian-pcx-14165.cloudflare-docs-7ou.pages.dev/durable-objects/platform/limits/

File: src/scripts/footnotes.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ if (footnotes) {
 		const content = note.querySelector("p") as HTMLParagraphElement;
 
 		const fnrefs = document.querySelectorAll<HTMLAnchorElement>(
-			`#${note.id.replace("fn", "fnref")}`,
+			`a[id^='${note.id.replace("fn", "fnref")}']`,
```
Comment (GregBrimble) [inline src/scripts/footnotes.ts:12]: Oh, does duplicate ones do something like `id="blah-2"`? Is that why you're switching to `^=`?

**Review by GregBrimble — APPROVED** [review]

File: src/scripts/footnotes.ts (hunk 1 above)
Comment (KianNH) [reply src/scripts/footnotes.ts:12]: Yep! You get `user-content-fnref-2` and `user-content-fnref-2-2` 😢

**Pull request merged**